lidiabrowser/sync/populate.py

```diff
@@ -17,11 +20,49 @@
     Publication,
     TermGroup,
 )
+from sync.zoteroutils import get_attachment_url, get_attachment_id_from_url
+
 
 logger = logging.getLogger(__name__)
 
 
 LIDIAPREFIX = "~~~~LIDIA~~~~"
+LEXICON_URLS = {}
+
+
+def fetch_lexicon_data():
+    url = "https://github.com/CentreForDigitalHumanities/lidia-zotero/raw/main/vocabulary/lexicon.xlsx"
+    filename = "../data/lexicon.xlsx"
+
+    if os.path.isfile(filename) and os.path.getsize(filename) > 0:
+        logger.info("Lexicon spreadsheet already downloaded.")
```

Comment:
There is no way to update the file now, but I think that is okay for the moment

Reply:
Right, since syncing is a manual step, deleting the local file is little trouble.